### ISSUE TITLE
[Feature Request] Add support for Horizontal Rule element in Requests lists

### ISSUE THREAD

**Issue by yarozar** [getinsomnia/insomnia#1773]
- Insomnia Version: 7.0.3.3074
- Operating System: macOS Catalina 10.15.1

## Details

Alongside "New Request" and "New Folder" would be nice to have something like "New Horizontal Rule" which will create, well, horizontal rule (`<hr />`) to help users better organize their requests visually. At the moment we have only folders for that purpose.

Thanks!

<img width="506" alt="1911100900_insomnia" src="https://user-images.githubusercontent.com/2981562/68540352-7d94ba00-0399-11ea-94b9-e593480216d8.png">

**Comment by elemanhillary** [comment]
Not detailed enough

**Comment by yarozar** [comment]
I have marked with red arrows examples of "horizontal rulers" that help organize browser menu. In HTML you use HR tag for that purpose:

<img width="651" alt="1911132041_insomnia" src="https://user-images.githubusercontent.com/2981562/68793739-32f39600-0656-11ea-804b-e8a837ecb3a3.png">

Or on GitHub page itself:
<img width="702" alt="1911132044_insomnia2" src="https://user-images.githubusercontent.com/2981562/68793888-87971100-0656-11ea-8631-4f7befb80d48.png">

**Comment by elemanhillary** [comment]
@yarozar i understand you now, what you need is to put the greyish line between new request and new folder

**Comment by yarozar** [comment]
@elemanhillary No, you are wrong. What I suggest is to give user opportunity to insert such "greyish line" (which is called "horizontal rule" actually) to divide his REST actions. LIke this-
<img width="519" alt="insom3" src="https://user-images.githubusercontent.com/2981562/68809608-f2efdb80-0674-11ea-9e3c-9e9f3ba61f91.png">

Please re-read my initial description. Thanks!

**Comment by elemanhillary** [comment]
@yarozar if you would suggest, after how many requests do you want the greyish line to appear.

**Comment by yarozar** [comment]
This is getting so complicating, amazing. :)

Currently there are 2 action menu items available - "New Request" and "New Folder" - one adds new request and another - folder. What I suggest is to add 3rd action item - "New Horizintal Rule" which will add horizontal ruler - "greyish line" - which I can drag and move as other requests or folder to make my APIs better organized. 

That's it. I don't know how I can explain this better.

**Comment by nijikokun** [comment]
I think I understand what @yarozar is saying here.

He'd like a new option in the dropdown: 

Add Separator

This raises a few questions:

1. Can you drag the separator around?
2. How do you delete the separator?

Outside of those, I could see it being useful. Does that properly represent the ask @yarozar ?

**Comment by yarozar** [comment]
@nijikokun , correct.

<img width="475" alt="insomnia" src="https://user-images.githubusercontent.com/2981562/70305630-8dbf7e00-180d-11ea-98e6-225db2e1daad.png">

re 1: Yes, you should be able to drag added separators in the list same way as you can drag standard Requests or Folders today.

re 2: Probably similar as for Requests or Folders in the same list - right clicking / context menu.

**Comment by gschier** [comment]
I'd like to understand the use case better. When would you use a separator instead of a folder? Can you provide some real world examples of when you would want to do this?

**Comment by yarozar** [comment]
@gschier - It would be an alternative to "folders way" of visually organizing your RESTs. I personally don't like folders at all because to get to the action inside folder you need one extra click.
